osquery/tables/system/posix/ssh_configs.cpp

```diff
@@ -0,0 +1,96 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <string>
+#include <vector>
+
+#include <boost/algorithm/string/trim.hpp>
+#include <boost/algorithm/string.hpp>
+
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/posix/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/tables/system/system_utils.h"
+
+namespace osquery {
+namespace tables {
+
+const std::string kUserSSHConfig = ".ssh/config";
+const std::string kSystemwideSSHConfig = "/etc/ssh/ssh_config";
+
+void genSSHConfig(const std::string& uid,
+                  const std::string& gid,
+                  const boost::filesystem::path& filepath,
```

Comment:
Feel free to declare `namespace fs = boost::filesystem;` above, and then you could have `const fs::path&`

Reply:
👍